docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -20,84 +20,97 @@ children:
   output: 'web, pdf'
   children:
 
-  - title: 'Todo Tutorial'
-    url: todo-tutorial.html
+  - title: 'Tutorials'
```

Comment:
I believe `lb4_sidebar.yml` is copied from loopback-next and any changes made here will be discarded when we run the next update. Please make the changes in the following file instead:
https://github.com/strongloop/loopback-next/blob/master/docs/site/sidebars/lb4_sidebar.yml

Reply:
I am working in the loopback-next repo - /Users/bschramm/loopback-next/docs/site is my local repo. Is that not the correct one?

Reply:
Weird. I believed I was reviewing a pull request in https://github.com/strongloop/loopback.io 😱 

I apologize for the confusion 🙈